gatsby-config.js

```diff
@@ -37,6 +37,13 @@ module.exports = {
         path: `${__dirname}/src/fonts/`,
       },
     },
+    {
+      resolve: `gatsby-plugin-goatcounter`,
+      options: {
+        code: 'the-hackathon',
+        referrer: true,
```

Comment:
Would you consider adding `head: false`? I can't tell if this is the default, but it moves the script to the body, speeding up the page load.

Reply:
it's the default. I took a close look at that when adding this to my own site